sdks/python/src/lilypad/_internal/otel/_utils.py

```diff
@@ -31,7 +31,7 @@ def client_is_already_instrumented(client: Any) -> bool:
 
 def mark_client_as_instrumented(client: Any) -> None:
```

Comment:
Wondering if this should be private too? Presumably this should get called when the client is actually instrumented. Calling it without actually instrumenting hte client seems like a setup for confusion.